docs/DeveloperGuide.md

```diff
@@ -288,31 +288,215 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+---
+**System**: `PoochPlanner`
+
+**Use case**: `UC01 - Add Contact of a Person`
```

Comment:
i think all use cases currently should have *** to indicate its a must have

Reply:
I think that is for user stories not use case.